### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by jaykayone** [Geoportail-Luxembourg/geoportailv3#588]
I just did a total rebuild on the integration server and now I am facing the following error:

```
mod_wsgi (pid=49773): Target WSGI script '/var/www/vhosts/geoportailv3/apache/geoportailv3.wsgi' cannot be loaded as Python module.
mod_wsgi (pid=49773): Exception occurred processing WSGI script '/var/www/vhosts/geoportailv3/apache/geoportailv3.wsgi'.
Traceback (most recent call last):
  File "/var/www/vhosts/geoportailv3/apache/geoportailv3.wsgi", line 11, in <module>
    application = get_app('/var/www/vhosts/geoportailv3/production.ini', 'main')
  File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/pyramid/paster.py", line 31, in get_app
    global_conf=options)
  File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 272, in loadobj
    return context.create()
  File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
    return self.object_type.invoke(self)
  File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 203, in invoke
    app = context.app_context.create()
  File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
    return self.object_type.invoke(self)
  File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 146, in invoke
    return fix_call(context.object, context.global_conf, **context.local_conf)
  File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/util.py", line 55, in fix_call
    val = callable(*args, **kw)
  File "/var/www/vhosts/geoportailv3/geoportailv3/__init__.py", line 94, in main
    config.scan()
  File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/pyramid/config/__init__.py", line 964, in scan
    ignore=ignore)
  File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/venusian/__init__.py", line 205, in scan
    __import__(modname)
  File "/var/www/vhosts/geoportailv3/geoportailv3/views/luxprintproxy.py", line 59, in <module>
    from c2cgeoportal.lib.caching import init_cache_control, get_region
ImportError: cannot import name init_cache_control
``

**Comment by jaykayone** [comment]
after the merge, I still get this when trying to print:

```
   __import__(modname)
 File "/var/www/vhosts/geoportailv3/geoportailv3/views/luxprintproxy.py", line 59, in <module>
   from c2cgeoportal.lib.caching import init_cache_control, get_region
mportError: cannot import name init_cache_control
I [py.warnings][MainThread] /var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/sqlahelper.py:47: SAWarning: At least on
ady been created.
gine)
 mod_wsgi (pid=60162): Target WSGI script '/var/www/vhosts/geoportailv3/apache/geoportailv3.wsgi' cannot be loaded as Python module.
 mod_wsgi (pid=60162): Exception occurred processing WSGI script '/var/www/vhosts/geoportailv3/apache/geoportailv3.wsgi'.
 Traceback (most recent call last):
   File "/var/www/vhosts/geoportailv3/apache/geoportailv3.wsgi", line 11, in <module>
     application = get_app('/var/www/vhosts/geoportailv3/production.ini', 'main')
   File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/pyramid/paster.py", line 31, in get_app
     global_conf=options)
   File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
     return loadobj(APP, uri, name=name, **kw)
   File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 272, in loadobj
     return context.create()
   File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
     return self.object_type.invoke(self)
   File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 203, in invoke
     app = context.app_context.create()
   File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
     return self.object_type.invoke(self)
   File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 146, in invoke
     return fix_call(context.object, context.global_conf, **context.local_conf)
   File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/paste/deploy/util.py", line 55, in fix_call
     val = callable(*args, **kw)
   File "/var/www/vhosts/geoportailv3/geoportailv3/__init__.py", line 94, in main
     config.scan()
   File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/pyramid/config/__init__.py", line 964, in scan
     ignore=ignore)
   File "/var/www/vhosts/geoportailv3/.build/venv/lib/python2.7/site-packages/venusian/__init__.py", line 205, in scan
     __import__(modname)
   File "/var/www/vhosts/geoportailv3/geoportailv3/views/luxprintproxy.py", line 59, in <module>
     from c2cgeoportal.lib.caching import init_cache_control, get_region
 ImportError: cannot import name init_cache_control
```

**Comment by jaykayone** [comment]
sorry my mistake. forget this

**Comment by sbrunner** [comment]
It wasn't a mistake, I fix it with #589 :-)
